bin/upscaler.py

```diff
@@ -166,84 +169,159 @@ def _upscale_frames(self):
             self.progress_bar_exit_signal = True
             progress_bar.join()
             return
+        elif self.waifu2x_driver == 'waifu2x_caffe':
+            # create a container for all upscaler threads
+            upscaler_threads = []
+
+            # list all images in the extracted frames
+            frames = [os.path.join(self.extracted_frames, f) for f in os.listdir(self.extracted_frames) if os.path.isfile(os.path.join(self.extracted_frames, f))]
```

Comment:
I am not a pythonista but isn't this a bit long for a single line? Maybe split into multiple lines?